lib/prestashopper/product.rb

```diff
@@ -1,21 +0,0 @@
-require 'active_support/core_ext/hash'
-
-module Prestashopper
-
-  # Has methods to convert the XML returned from the API to a ruby hash
-  class Product
```

Comment:
This class is no longer needed using `method_missing`, and parsing is now completed during initialization.